recipe/activate.sh

```diff
@@ -1,10 +1,26 @@
 #!/bin/bash
-if [ "$(uname)" == "Darwin" ]
-then
-    # for Mac OSX
-    export KERAS_BACKEND=tensorflow
-elif [ "$(uname)" == "Linux" ]
-then
-    # for Linux
-    export KERAS_BACKEND=theano
+
+# get the location of the keras.json file
+KERAS_JSON="$HOME/.keras/keras.json"
+
+if [ -f $KERAS_JSON ]; then # if the file exists
+
+  BACKEND_KEY="backend" # value we are interested in finding
+  KEY_VALUE_REGEX="\"($BACKEND_KEY)\": \"([^\"]*)\"" # regex
+  JSON_TEXT=$(cat $KERAS_JSON) # data in keras.json file
+
+  if [[ $JSON_TEXT =~ $KEY_VALUE_REGEX ]]; then # check for a match
+    BACKEND_VALUE="${BASH_REMATCH[2]}" # get the user defined backend value
+    export KERAS_BACKEND=$BACKEND_VALUE # set the environmental variable
+  fi
```

Comment:
The activation script was a rather dirty hack for making sure that the keras version conda-forge ships works by default. The problem is (was?) that some backend combinations did not work because the tensorflow package did not exist or was broken for some arches.
I think the simplest way would be to get tensorflow right and get rid of the activation scripts again.